chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -136,7 +136,7 @@
           hostname mnode.name
           macaddress mac_list[i]
           if mnode.macaddress == mac_list[i]
-            ipaddress admin_data_net.address
+            ipaddress admin_data_net.address unless admin_data_net.nil?
```

Comment:
It has 192.168.126.24 in my test, which is in the correct range, installed nodes have .81+